demos/launch/airport_terminal.launch.xml

```diff
@@ -58,4 +58,30 @@
     </include>
   </group>
 
+    <!-- Caddy spawner, fleet adapter and robot state aggregator needed for Caddy readonly_plugin -->
+  <group>
+    <let name="fleet_name" value="caddy"/>
+    <let name="caddy_path" value="$(find-pkg-share rmf_demo_assets)/models/Caddy/model.sdf" />
+    <node pkg="gazebo_ros" exec="spawn_entity.py" args="-entity caddy -file $(var caddy_path) -x 64.08 -y -23.53 -Y 1.57">
+        <param name="use_sim_time" value="true"/>
+    </node>
+    <include file="$(find-pkg-share demos)/include/adapters/caddy_adapter.launch.xml">
+    <arg name="use_sim_time" value="true"/>
+    <arg name="fleet_name" value="$(var fleet_name)"/>
```

Comment:
tiny nitpick, indentation for L69 and L70

Reply:
Good catch